### PR TITLE
IDE-818 ECL IDE unresponsive on login

### PR DESCRIPTION
**Pull request by dehilsterlexis**
Fixed IDE-818

Signed-off-by: dehilsterlexis <david.dehilster@lexisnexis.com>

### CODE REVIEW

**Comment by dehilsterlexis** [comment]
@GordonSmith added a preference checkbox button that is default off for loading meta-data. For review.

File: comms/EclCC.cpp [hunk 1]
```diff
@@ -11,6 +11,7 @@
 #include "SMC.h"
 #include <UtilFilesystem.h>
 #include "DiskAttribute.h"
+#include <../wlib/PreferenceDlg.h>
```
Comment (GordonSmith) [inline comms/EclCC.cpp]: Comms can not have a dependency on wlib....

File: en_us/en_us.rc [hunk 2]
```diff
@@ -372,6 +372,7 @@ BEGIN
     CONTROL         "Double Click Selects Qualified Label",IDC_CHECK_DOUBLECLICKSEL,
                     "Button",BS_AUTOCHECKBOX | WS_TABSTOP,71,135,127,10
     CONTROL         "Target Background Colors",IDC_CHECK_TARGETCOLOR,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,71,93,97,10
+    CONTROL         "Meta Data",IDC_CHECK_METADATA,"Button",BS_AUTOCHECKBOX | BS_ICON | WS_TABSTOP,227,68,49,10
```
Comment (GordonSmith) [inline en_us/en_us.rc]: I don't think I see this as an editor option - more a "compiler" option.

**Comment by dehilsterlexis** [comment]
@GordonSmith for review

**Pull request merged**